raster/r.in.lidar/info.c

```diff
@@ -103,7 +103,9 @@ int scan_bounds(LASReaderH LAS_reader, int shell_style, int extents, int update,
     first = TRUE;
 
     /* init to nan in case no points are found */
-    min_x = max_x = min_y = max_y = min_z = max_z = 0.0 / 0.0;
+	double nan_val;
+	sscanf("-nan", "%lf", &nan_val);
+    min_x = max_x = min_y = max_y = min_z = max_z = nan_val;
```

Comment:
In #2681, it has been changed to use `NAN` like this:
```c
    /* init to nan in case no points are found */
    min_x = max_x = min_y = max_y = min_z = max_z = NAN;
```